com.pot_app.pot.yml

```diff
@@ -0,0 +1,156 @@
+id: com.pot_app.pot
+runtime: org.gnome.Platform
+runtime-version: '45'
+sdk: org.gnome.Sdk
+command: pot
+rename-icon: pot
+rename-desktop-file: pot.desktop
+
+finish-args:
+  - --socket=x11
+  - --socket=pulseaudio
+  - --share=ipc
+  - --share=network
+  - --device=dri
+  - --talk-name=org.freedesktop.Notifications
```

Comment:
Is this really needed? It should go through the portal

Reply:
I have removed
```  - --talk-name=org.freedesktop.Notifications```